core/src/main/scala/kafka/server/metadata/KRaftMetadataCache.scala

```diff
@@ -272,7 +272,7 @@ class KRaftMetadataCache(
    * @param maximumNumberOfPartitions     The max number of partitions to return.
    * @param ignoreTopicsWithExceptions    Whether ignore the topics with exception.
    */
```

Comment:
Move this scaladoc to the interface.

Reply:
@FrankYang0529 Could you please remove those comments as you have moved them to the interface